react/src/pages/VFolderNodeListPage.tsx

```diff
@@ -306,7 +312,11 @@ const VFolderNodeListPage: React.FC<VFolderNodeListPageProps> = ({
                 style={{ height: lg ? 200 : undefined }}
                 fetchKey={deferredFetchKey}
                 onRequestBadgeClick={() => {
-                  setIsInvitationResponseModalOpen(true);
+                  webuiNavigate({
+                    search: new URLSearchParams({
+                      invitation: '1',
```

Comment:
What's the meaning of input `1` as a value of invitation key?

Reply:
I changed it to StringParam for better understanding. @lizable